tests/components/modbus/test_modbus_binary_sensor.py

```diff
@@ -31,13 +41,7 @@ async def test_config_binary_sensor(hass, do_discovery, do_options):
         CONF_ADDRESS: 51,
     }
     if do_options:
-        config_sensor.update(
-            {
-                CONF_SLAVE: 10,
-                CONF_INPUT_TYPE: CALL_TYPE_DISCRETE,
-                CONF_DEVICE_CLASS: "door",
-            }
-        )
+        config_sensor.update()
```

Comment:
We should just update with `do_options` right?

Reply:
you are quite right! Corrected and while I was add it, I changed "config_sensor.update(do_options)" to use "**do_options", that seems easier to read.

Thanks for catching this one.